django_dicom/views.py

```diff
@@ -145,17 +143,6 @@ def pixel_data(self, request, pk: int = None):
             return Response(series.get_data(as_json=True))
         return Response("Invalid series ID!", status=status.HTTP_404_NOT_FOUND)
 
-    @action(detail=True)
```

Comment:
Exactly. Even if we will need this kind of functionality in the future, now that we're using Vuex it would probably make more sense to simply implement it as a [getter method](https://vuex.vuejs.org/guide/getters.html).